services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_core.py

```diff
@@ -97,6 +94,10 @@ def _node_not_ready(node: Node) -> bool:
         else:
             pending_nodes.append(instance)
 
+    def _node_not_ready(node: Node) -> bool:
```

Comment:
Q: Why to not define it outside of function?

Reply:
no specific reason, was already like this before. moved.